### DISCUSSION TITLE
Can not run prisma-example on Win10

### DISCUSSION THREAD

**Discussion by jjjuk** [prisma/prisma#2318] [Q&A]
### Hello, might somebody help me? I am beginner in developing and I can't understand what is going wrong.

prisma-examples/typescript/rest-next.js

installation throws an error:

 > @prisma/client@2.0.0-beta.2 postinstall C:\Users\gmodh\Desktop\JENYA\DEV\PRISMA EXAMPLE\prisma-examples\typescript\rest-nextjs\node_modules\@prisma\client
 > node scripts/postinstall.js
 
> "C:\Users\gmodh\Desktop\JENYA\DEV\PRISMA" �� ���� ����७��� ��� ���譥�
> ��������, �ᯮ��塞�� �ணࠬ��� ��� ������ 䠩���.
> Error: Cannot call write after a stream was destroyed

and then after `npm run dev`:

> Error: @prisma/client did not initialize yet. Please run "prisma generate" and try to import it again.
> In case this error is unexpected for you, please report it in https://github.com/prisma/prisma-client-js/issues/390.
>     at new PrismaClient (C:\Users\gmodh\Desktop\JENYA\DEV\PRISMA EXAMPLE\prisma-examples\typescript\rest-nextjs\node_modules\@prisma\client\index.js:3:11)
>     at Module../pages/api/feed.ts (C:\Users\gmodh\Desktop\JENYA\DEV\PRISMA EXAMPLE\prisma-examples\typescript\rest-nextjs\.next\server\static\development\pages\api\feed.js:109:16)   
>     at __webpack_require__ (C:\Users\gmodh\Desktop\JENYA\DEV\PRISMA EXAMPLE\prisma-examples\typescript\rest-nextjs\.next\server\static\development\pages\api\feed.js:23:31)
>     at Object.5 (C:\Users\gmodh\Desktop\JENYA\DEV\PRISMA EXAMPLE\prisma-examples\typescript\rest-nextjs\.next\server\static\development\pages\api\feed.js:131:18)
>     at __webpack_require__ (C:\Users\gmodh\Desktop\JENYA\DEV\PRISMA EXAMPLE\prisma-examples\typescript\rest-nextjs\.next\server\static\development\pages\api\feed.js:23:31)
>     at C:\Users\gmodh\Desktop\JENYA\DEV\PRISMA EXAMPLE\prisma-examples\typescript\rest-nextjs\.next\server\static\development\pages\api\feed.js:91:18
>     at Object.<anonymous> (C:\Users\gmodh\Desktop\JENYA\DEV\PRISMA EXAMPLE\prisma-examples\typescript\rest-nextjs\.next\server\static\development\pages\api\feed.js:94:10)
>     at Module._compile (internal/modules/cjs/loader.js:1157:30)
>     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1177:10)
>     at Module.load (internal/modules/cjs/loader.js:1001:32)
>     at Function.Module._load (internal/modules/cjs/loader.js:900:14)
>     at Module.require (internal/modules/cjs/loader.js:1043:19)
>     at require (internal/modules/cjs/helpers.js:77:18)
>     at DevServer.handleApiRequest (C:\Users\gmodh\Desktop\JENYA\DEV\PRISMA EXAMPLE\prisma-examples\typescript\rest-nextjs\node_modules\next\dist\next-server\server\next-server.js:455:28)
>     at async Object.fn (C:\Users\gmodh\Desktop\JENYA\DEV\PRISMA EXAMPLE\prisma-examples\typescript\rest-nextjs\node_modules\next\dist\next-server\server\next-server.js:386:37)       
>     at async Router.execute (C:\Users\gmodh\Desktop\JENYA\DEV\PRISMA EXAMPLE\prisma-examples\typescript\rest-nextjs\node_modules\next\dist\next-server\server\router.js:134:32)       
> FetchError: invalid json response body at http://localhost:3000/api/feed reason: Unexpected token I in JSON at position 0
>     at C:\Users\gmodh\Desktop\JENYA\DEV\PRISMA EXAMPLE\prisma-examples\typescript\rest-nextjs\node_modules\node-fetch\lib\index.js:272:32
>     at runMicrotasks (<anonymous>)
>     at processTicksAndRejections (internal/process/task_queues.js:97:5)
>     at async getServerSideProps (C:\Users\gmodh\Desktop\JENYA\DEV\PRISMA EXAMPLE\prisma-examples\typescript\rest-nextjs\.next\server\static\development\pages\index.js:2127:16)       
>     at async Object.renderToHTML (C:\Users\gmodh\Desktop\JENYA\DEV\PRISMA EXAMPLE\prisma-examples\typescript\rest-nextjs\node_modules\next\dist\next-server\server\render.js:341:24)  
>     at async DevServer.renderToHTMLWithComponents (C:\Users\gmodh\Desktop\JENYA\DEV\PRISMA 
> EXAMPLE\prisma-examples\typescript\rest-nextjs\node_modules\next\dist\next-server\server\next-server.js:652:26)
>     at async DevServer.renderToHTML (C:\Users\gmodh\Desktop\JENYA\DEV\PRISMA EXAMPLE\prisma-examples\typescript\rest-nextjs\node_modules\next\dist\next-server\server\next-server.js:799:28)
>     at async DevServer.renderToHTML (C:\Users\gmodh\Desktop\JENYA\DEV\PRISMA EXAMPLE\prisma-examples\typescript\rest-nextjs\node_modules\next\dist\server\next-dev-server.js:19:539)  
>     at async DevServer.render (C:\Users\gmodh\Desktop\JENYA\DEV\PRISMA EXAMPLE\prisma-examples\typescript\rest-nextjs\node_modules\next\dist\next-server\server\next-server.js:541:22)    at async Object.fn (C:\Users\gmodh\Desktop\JENYA\DEV\PRISMA EXAMPLE\prisma-examples\typescript\rest-nextjs\node_modules\next\dist\next-server\server\next-server.js:391:17)       
>     at async Router.execute (C:\Users\gmodh\Desktop\JENYA\DEV\PRISMA EXAMPLE\prisma-examples\typescript\rest-nextjs\node_modules\next\dist\next-server\server\router.js:134:32)       
>     at async DevServer.run (C:\Users\gmodh\Desktop\JENYA\DEV\PRISMA EXAMPLE\prisma-examples\typescript\rest-nextjs\node_modules\next\dist\next-server\server\next-server.js:506:29)   
>     at async DevServer.handleRequest (C:\Users\gmodh\Desktop\JENYA\DEV\PRISMA EXAMPLE\prisma-examples\typescript\rest-nextjs\node_modules\next\dist\next-server\server\next-server.js:150:20) {
>   **message: 'invalid json response body at http://localhost:3000/api/feed reason: Unexpected token I in JSON at position 0',
>   type: 'invalid-json'**
> }

App isn't working. 

42 | export const getServerSideProps: GetServerSideProps = async () => {
43 |   const res = await fetch('http://localhost:3000/api/feed')
**44 |   const feed = await res.json()** //error in this line
45 |   return {
46 |     props: { feed },
47 |   }

**Comment by jjjuk** [comment]
And the same for javascript example, react-apollo-serever even cant run `npm run dev`

**Comment by ryands17** [comment]
Hey @jjjuk :wave: 
Do you have any global installation of `prisma`? If so, could you unisntall that and try again.

Also which version of Node do you have currently?
